ci/run_python_tests.sh

```diff
@@ -2,12 +2,6 @@
 
 set -euo pipefail
 
-test_cmd="${1:-}"
-test_args="${2:-}"
```

Comment:
I think you should leave check for 2 arguments: test_cmd and test_args. You do not need node version. 
Why did you decided to get rid of it?